docker-compose-generator/docker-fragments/btcpayserver.yml

```diff
@@ -4,7 +4,7 @@ services:
 
   btcpayserver:
     restart: unless-stopped
-    image: nicolasdorier/btcpayserver:1.0.3.9
+    image: ${BTCPAY_IMAGE:-nicolasdorier/btcpayserver:nicolasdorier/btcpayserver:1.0.3.9}
```

Comment:
it broke `${BTCPAY_IMAGE:-nicolasdorier/btcpayserver:nicolasdorier/btcpayserver:1.0.3.9}` double tag